_commands/ubuntu/poweroff.md

```diff
@@ -0,0 +1,73 @@
+---
+---
+
+poweroff
+--------
+
+`poweroff` is used allow a system administrator to poweroff the system
+
+~~~ bash
+$ poweroff [options]
+~~~
+
+
+<!--more-->
+
+### Details
+ * When called with --force or when in runlevel 0 or 6, this tool invokes the
+   reboot system call itself and directly reboots the system.
+ * Otherwise, this simply invokes the shutdown tool with the appropriate
+   arguments without passing REBOOTCOMMAND argument.
+
+### Useful Options
+
+#### `poweroff --f`
+#### `poweroff --force`
+
+~~~bash
+$ poweroff -f
+~~~
+
+ * Does not invoke shutdown, and instead performs the actual command you would expect from the name.
```

Comment:
This is incredibly ambiguous and you should clarify what the "actual command you would expect from the name" is.